js_src/_dart_helpers.js

```diff
@@ -131,7 +137,7 @@ function _createReactDartComponentClass2(dartInteropStatics, componentStatics, j
   }
 
   // Delete methods that the user does not want to include (such as error boundary event).
-  jsConfig.skipMethods.forEach((method) => {
+  jsConfig.skipMethods.forEach(method => {
```

Comment:
I'm unfamiliar with this syntax - is it kosher to ditch the parentheses? 

Out of curiosity, what code change on the Dart side resulted in this change?

Reply:
Yeah it it's kosher, because it gets babelified this get de-sugared. its syntactic sugar for that is part of ES6, if there is only a single argument, you can even leave off the parenthesis around the argument, it's mainly just formatting change, nothing functionally besides `_jsNull` changed here.